From 16b28d9cd002a4b16727e5c48b074d8826c2733b Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Tue, 28 Jul 2020 13:46:23 +0700
Subject: [PATCH 380/443] media: dt-bindings: rcar-csi2: Add r8a774a3

Add the compatible string for RZ/G2M v3.0 (a.k.a. R8A774A3) to the
list of supported SoCs.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 Documentation/devicetree/bindings/media/renesas,rcar-csi2.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/media/renesas,rcar-csi2.txt b/Documentation/devicetree/bindings/media/renesas,rcar-csi2.txt
index e3286fb..178531b 100644
--- a/Documentation/devicetree/bindings/media/renesas,rcar-csi2.txt
+++ b/Documentation/devicetree/bindings/media/renesas,rcar-csi2.txt
@@ -9,6 +9,7 @@ Mandatory properties
 --------------------
  - compatible: Must be one or more of the following
    - "renesas,r8a774a1-csi2" for the R8A774A1 device.
+   - "renesas,r8a774a3-csi2" for the R8A774A3 device.
    - "renesas,r8a774b1-csi2" for the R8A774B1 device.
    - "renesas,r8a774c0-csi2" for the R8A774C0 device.
    - "renesas,r8a774e1-csi2" for the R8A774E1 device.
-- 
2.7.4

